codingTest_jye/todolist/app/src/models/UserStorage.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const db = require("../config/db");
+
+class UserStorage {
+  static async getDescription() {
+    return new Promise((resolve, reject) => {
+      const query = "SELECT * FROM tdList";
+      db.query(query, (err, data) => {
+        if (err) {
+          reject(`${err}`);
+        } else {
+          // console.log("UserStorage.js의 동작");
+          // console.log(data);
+          resolve(data);
+        }
```

Comment:
```suggestion
        } 
          // console.log("UserStorage.js의 동작");
          // console.log(data);
          resolve(data);
```
Promise는 어차피 reject 아니면 resolve가 호출되기 때문에 else가 없어도 될 것 같습니다.
밑에서 모든 코드 또한 마찬가지 입니다.

Reply:
알겠습니다!!